tests/unittests/test_command_line_processing.py

```diff
@@ -59,11 +60,24 @@ def test_non_existent_session_file_returns_EX_NOINPUT(local_unittest_config):
     assert cosmic_ray.cli.main(["exec", str(local_unittest_config), "foo.session"]) == ExitCode.NO_INPUT
 
 
-def test_non_existent_config_file_returns_EX_NOINPUT(session, local_unittest_config):
-    cosmic_ray.cli.main(["init", local_unittest_config, str(session)])
+def test_non_existent_config_file_returns_EX_NOINPUT(session, local_unittest_config,force):
```

Comment:
This test (and all tests which take a `force` parameter) don't work because you don't have a fixture providing the value for `force`. In all cases, I don't think the tests actually need to use a fixture for `force`. You can simply pass the correct value - `True` or `False` - into `cosmic_ray.cli.main`.